WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Free to Paid Plans/FreeToPaidPlansCoordinator.swift

```diff
@@ -2,6 +2,9 @@ import UIKit
 import SwiftUI
 
 @objc final class FreeToPaidPlansCoordinator: NSObject {
+
+    typealias PurchaseCallBack = ((UIViewController, String) -> Void)
```

Comment:
[Nit]
```suggestion
    typealias PurchaseCallback = ((UIViewController, String) -> Void)
```

Reply:
Resolved in b6486a741d121e606866b8caecdb7e7d8978b11b 👍